### ISSUE TITLE
No way to select fleet and equipment for work orders

### ISSUE THREAD

**Issue by Andrevdb** [landinfotech/civitas-asset-management#179]
@meomancer @timlinux @tjaartvdb 

Tjaart and I discovered something we totally forgot about during this. There are some non-spatial features in the fleet and equipment layer need to be able to be selected as features to add to the work order. 

Maybe when the Fleet and Equipment group is toggled on/off on the legend, a box could pop up with the fleet and equipment features, in table form? Sometimes the Equipment is linked to a parent feature. Example: a pump/motor belonging to the water pumphouse structure. I think a good way to navigate the fleet and equipment list is by using a table that we can filter.

My apologies Irwan, this is super late for this.

**Comment by Andrevdb** [comment]
Do you think there is time/budget to add a table view of the displayed features from the legend in this project? 
![image](https://user-images.githubusercontent.com/40511773/132404330-f182a50e-539d-42a7-922a-453e2969be8b.png)

**Comment by tjaartvdb** [comment]
Yes, we definitely need this functionality. If we can not do it in the Sept release, we should add it to the next version. I will connect with @timlinux re available budget

**Comment by tjaartvdb** [comment]
@timlinux @meomancer @Andrevdb 
Will it be possible to add this functionality to the Oct release? It was identified as a functionality required by ToPH

**Comment by Andrevdb** [comment]
Moved this out of the issues for this summer's project.
